package_check.sh

```diff
@@ -24,7 +24,7 @@ print_help() {
     -S, --storage-dir DIRECTORY Where to store temporary test files like yunohost backups
     -h, --help                  Display this help
 
-    Pass YNHDEV_BACKEND=incus to use incus instead of lxd.
+    Pass YNHDEV_BACKEND=incus|lxd to use a specific LXD-compatible backend.
```

Comment:
```suggestion
    Pass YNHDEV_BACKEND=incus|lxc to use a specific LXD-compatible backend.
```

According to the `if` in common.sh ? Or do we really want `lxd` ?

Reply:
Nope, the `$lxc` variable is for the CLI name (incus or lxc) , but the actual system/daemon name is LXD. `YNHDEV_BACKEND=lxc` should remain available if LXC (not LXD) is ever supported.

Reply:
I'm refering to https://github.com/YunoHost/package_check/blob/2490e357313b28da7cc0bec882d57f7a576ebdf9/lib/common.sh#L38 and https://github.com/YunoHost/package_check/blob/2490e357313b28da7cc0bec882d57f7a576ebdf9/lib/lxc.sh#L11 which suggests the var's meaninful values are `incus` and `lxc`, not `lxd` ..?

Reply:
Yes exactly because LXD uses "lxc" binary name. Yes that's confusing but `lxc` command has nothing to do with LXC. u_u

Reply:
So `$lxc` can be either `incus`, `lxd` or `lxc` ...?

Reply:
`$lxc` is the CLI name. For the moment only `incus` or `lxc` but in that case `lxc` is for LXD not LXC. That cannot be changed without massive changes in the code.